dafni_cli/models/upload.py

```diff
@@ -34,6 +34,10 @@ def upload_model(
                                    upload a new model.
         json (bool): Whether to print the raw json returned by the DAFNI API
     """
+    if not is_valid_image_file(image_path):
+        click.echo("Invalid image file format")
```

Comment:
This message should maybe be a bit more decriptive
```suggestion
        click.echo("Your model image file type is incorrect. Please check you've entered the correct file and try again")
```